auth/src/main/java/org/openedx/auth/presentation/restore/RestorePasswordFragment.kt

```diff
@@ -269,12 +270,20 @@ private fun RestorePasswordScreen(
                                     description = stringResource(id = authR.string.auth_example_email),
                                     onValueChanged = {
                                         email = it
+                                        isEmailError = email.isEmpty()
```

Comment:
On value change, the error should be removed.

```suggestion
                                        isEmailError = false
```